custom_components/frigate/__init__.py

```diff
@@ -141,6 +144,16 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except FrigateApiClientError as exc:
         raise ConfigEntryNotReady from exc
 
+    if server_version < FRIGATE_VERSION_ERROR_CUTOFF:
```

Comment:
Nit: That's going to end up going a lexical comparison of the strings rather than a version comparison. That's probably okay here, but better off wrapping it in `AwesomeVersion` for best practice ([example](https://github.com/home-assistant/core/blob/dev/homeassistant/components/hyperion/__init__.py#L156)).